source/TestAdapter/Executor.cs

```diff
@@ -737,7 +737,7 @@ private void CheckAllTests(string rawOutput, List<TestResult> results)
                     // Format is "Test passed: MethodName, ticks";
                     // We do get split with space if the coma is missing, happens time to time
 
-                    string method = line.Substring(line.IndexOf(TestPassed) + TestPassed.Length).Split(',')[0].Split(' ')[0];
+                    string method = line.Substring(line.IndexOf(TestPassed) + TestPassed.Length).Split(',')[0];
```

Comment:
why do you remove the split? the comment explains why it is necessary to keep it

Reply:
With it it removes the "parameters" section of name which is required later to see exactly which test failed/passed. I've implemented it in a way that you can check from VS test runner exactly which parameters are failing.
Example here:
https://dotnetfiddle.net/XyJ5cz

Reply:
Also I've check runner source code and I can't see how the comma can be missing. Maybe it's obsolete?